### PR TITLE
[DVCSMP-2607] Standardize remaining SmartThingsPublic DTH colors

### PR DESCRIPTION
**Pull request by jackchi**
Change `ON/CLOSED/TAMPERED` from Green to Blue #00A0DC

Change `OPEN` from lighter orange to darker orange #e86d13

Changed Temperature levels colors

Please review @parijatdas @ShunmugaSundar 
cc @skt123 @pchomal @workingmonk @KyleLeneau

### CODE REVIEW

File: devicetypes/johnrucker/coopboss-h3vx.src/coopboss-h3vx.groovy [hunk 1]
```diff
@@ -135,13 +135,13 @@ metadata {
     	}        
         
 		standardTile("autoClose", "device.autoCloseEnable", width: 2, height: 2){
-			state "on", label: 'Auto', action:"autoCloseOff", icon: "st.doors.garage.garage-closing", backgroundColor: "#79b821", nextState:"Sent"
+			state "on", label: 'Auto', action:"autoCloseOff", icon: "st.doors.garage.garage-closing", backgroundColor: "#00A0DC", nextState:"Sent"
 			state "off", label: 'Auto', action:"autoCloseOn", icon: "st.doors.garage.garage-closing", nextState:"Sent"
 			state "Sent", label: 'wait', icon: "st.motion.motion.active", backgroundColor: "#ffa81e"
```
Comment (parijatdas) [inline devicetypes/johnrucker/coopboss-h3vx.src/coopboss-h3vx.groovy]: should the backgroundColor be changed from #ffa81e to #e86d13?

File: devicetypes/johnrucker/coopboss-h3vx.src/coopboss-h3vx.groovy [hunk 2]
```diff
@@ -135,13 +135,13 @@ metadata {
...
+			state "on", label: 'Auto', action:"autoCloseOff", icon: "st.doors.garage.garage-closing", backgroundColor: "#00A0DC", nextState:"Sent"
 			state "off", label: 'Auto', action:"autoCloseOn", icon: "st.doors.garage.garage-closing", nextState:"Sent"
 			state "Sent", label: 'wait', icon: "st.motion.motion.active", backgroundColor: "#ffa81e"              
 		}    
         
 		standardTile("autoOpen", "device.autoOpenEnable", width: 2, height: 2){
-			state "on", label: 'Auto', action:"autoOpenOff", icon: "st.doors.garage.garage-opening", backgroundColor: "#79b821", nextState:"Sent"
+			state "on", label: 'Auto', action:"autoOpenOff", icon: "st.doors.garage.garage-opening", backgroundColor: "#00A0DC", nextState:"Sent"
 			state "off", label: 'Auto', action:"autoOpenOn", icon: "st.doors.garage.garage-opening", nextState:"Sent"
 			state "Sent", label: 'wait', icon: "st.motion.motion.active", backgroundColor: "#ffa81e"
```
Comment (parijatdas) [inline devicetypes/johnrucker/coopboss-h3vx.src/coopboss-h3vx.groovy]: should the backgroundColor be changed from #ffa81e to #e86d13?

File: devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy [hunk 3]
```diff
@@ -52,20 +52,20 @@ metadata {
...
-				[value: 75, color: "#f0d030"], // yellow
-				[value: 80, color: "#fbf020"],
-				[value: 85, color: "#fbdc01"],
-				[value: 90, color: "#fb3a01"],
-				[value: 95, color: "#fb0801"]  // red=hot
+				[value: 0,  color: "#153591"], // blue=cold
+				[value: 65, color: "#153591"],
+				[value: 70, color: "#1e9cbb"],
+				[value: 75, color: "#90d2a7"],
+				[value: 80, color: "#44b621"], // green
```
Comment (workingmonk) [inline devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy]: is this part of the spec? why is 80F green? and 85F yellow

**Comment by jackchi** [comment]
Can you comment on the spec of the temperature levels? This DTH doesn't
follow the other temp scale. Perhaps I should just leave it as is?

Jack Chi, 415-216-3240
Software Engineer

**Comment by workingmonk** [comment]
@jackchi if there is no specification in the doc that Phoebe made, then don't change the colors.

**Comment by jackchi** [comment]
Confirmed with Phoebe on the temperature change of `ezmultipli.groovy` color temperature change.

**Review by workingmonk — APPROVED** [review]

**Pull request merged**